### PR TITLE
Dissolve out of Topics

### PR DESCRIPTION
**Pull request by multimokia**
Monika now dissolves out of topics/idle.

Already tested by Simple + Legend for functionality + any seizures, just need final says from them.

### CODE REVIEW

File: Monika After Story/game/event-handler.rpy [hunk 1]
```diff
@@ -2032,6 +2032,13 @@ label prompt_menu:
         if cb_label is not None:
             call expression cb_label
 
+        # NOTE: we only need to enable music hotkey since we are in dlg mode
+        #$ mas_DropShield_idle()
+        $ store.mas_hotkeys.music_enabled = True
```
Comment (ThePotatoGuy) [inline Monika After Story/game/event-handler.rpy]: why was this moved here, this should be last to prevent interruptions in flow

Comment (multimokia) [reply Monika After Story/game/event-handler.rpy]: Yeah, I moved it as during testing it took significantly longer to change the shields.

Comment (multimokia) [reply Monika After Story/game/event-handler.rpy]: Actually, time difference wasn't noticeable between the two, will swap back

**Comment by multimokia** [comment]
Also, I'd tested this with @Legendkiller21 and @MisterSimple before PRing to ensure no glitchiness between transitions. Their testing had shown none of that, hence why I'd PR'd it in the first place.

**Comment by Legendkiller21** [comment]
Apart from a slight delay of 0.5-1 sec in the transition between the shields, no glitched transitions have been observed during our testing.
